guides/v2.2/install-gde/trouble/php/tshoot_php-set.md

```diff
@@ -53,7 +58,9 @@ See [During installation, xdebug maximum function nesting level error]({{ page.b
 
 Error text is typically:
 
-    Parse error: syntax error, unexpected 'data' (T_STRING)
+```text
```

Comment:
```suggestion
```terminal
```

Reply:
Manually applied change to local branch since Im making other local changes.